composer.json

```diff
@@ -10,5 +10,10 @@
     "require": {
         "php": ">=5.5",
         "zendframework/zendframework": "~2.5"
+    },
+    "autoload": {
+        "psr-0": {
```

Comment:
You can use `psr-4` here, by simply configuring `"Application\\": "module/Application/src/Application"`